boards/arm/thingy91x_nrf9161/Kconfig.defconfig

```diff
@@ -0,0 +1,124 @@
+# Thingy:91 nRF9161 board configuration
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+if BOARD_THINGY91X_NRF9161 || BOARD_THINGY91X_NRF9161_NS
+
+DT_COMPAT_NORDIC_NRF700X_SPI := nordic,nrf700x-spi
+config NRF7002_ON_SPI
+	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF700X_SPI))
+
+config BOARD
+	default "thingy91x_nrf9161"
+
+# By default, if we build for a Non-Secure version of the board,
+# enable building with TF-M as the Secure Execution Environment.
+config BUILD_WITH_TFM
+	default y if BOARD_THINGY91X_NRF9161_NS
+
+if BUILD_WITH_TFM
+
+# By default, if we build with TF-M, instruct build system to
+# flash the combined TF-M (Secure) & Zephyr (Non Secure) image
+config TFM_FLASH_MERGED_BINARY
+	bool
+	default y
+
+config PM_PARTITION_SIZE_TFM_SRAM
+	hex
+	default 0xa000
+
+endif # BUILD_WITH_TFM
+
+# For the secure version of the board the firmware is linked at the beginning
+# of the flash, or into the code-partition defined in DT if it is intended to
+# be loaded by MCUboot. If the secure firmware is to be combined with a non-
+# secure image (TRUSTED_EXECUTION_SECURE=y), the secure FW image shall always
+# be restricted to the size of its code partition.
+# For the non-secure version of the board, the firmware
+# must be linked into the code-partition (non-secure) defined in DT, regardless.
+# Apply this configuration below by setting the Kconfig symbols used by
+# the linker according to the information extracted from DT partitions.
+
+if BOARD_THINGY91X_NRF9161 && TRUSTED_EXECUTION_SECURE
+
+config FLASH_LOAD_SIZE
+	default $(dt_chosen_reg_size_hex,$(DT_CHOSEN_Z_CODE_PARTITION))
+
+endif # BOARD_THINGY91X_NRF9161 && TRUSTED_EXECUTION_SECURE
+
+if BOARD_THINGY91X_NRF9161_NS
+
+config FLASH_LOAD_OFFSET
+	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_CODE_PARTITION))
+
+config FLASH_LOAD_SIZE
+	default $(dt_chosen_reg_size_hex,$(DT_CHOSEN_Z_CODE_PARTITION))
+
+endif # BOARD_THINGY91X_NRF9161_NS
+
+config BT_HCI_VS
+	default y if BT
+
+config HW_STACK_PROTECTION
+	default ARCH_HAS_STACK_PROTECTION
+
+# check if we are building for any bootloader
+if (!IS_SECURE_BOOTLOADER && !MCUBOOT)
```

Comment:
Please update to use `IS_BOOTLOADER_IMG` instead (see comment in thingy91x_nrf5340 folder).